types/intl-unofficial-duration-unit-format/tslint.json

```diff
@@ -0,0 +1,12 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "npm-naming": [
+            true,
+            {
+                "mode": "code",
+                "errors": [["NeedsExportEquals", false]]
+            }
+        ]
+    }
+}
```

Comment:
```suggestion
{
    "extends": "dtslint/dt.json"
}
```

Reply:
Yeah... The packages ships both ES6 and CJS modules. dtslint forces to use CJS exports, which is also a mistake.

Any ideas 🤔?

Reply:
It will work with default `esMduleInterop`:
```ts
import DurationUnitFormat from 'intl-unofficial-duration-unit-format';

const duration = new DurationUnitFormat();
const parts = duration.formatToParts(90);

console.log(parts);
```
compiles and runs fine.

Not that this package has invalid `package.json` declaration and needs to be updated itself:
```bash
Error [ERR_INVALID_PACKAGE_TARGET]: Invalid "exports" main target "dist/cjs/duration-unit-format.cjs.js" defined in the package config C:\develop\tests\node_modules\intl-unofficial-duration-unit-format\package.json; targets must start with "./"
```

Reply:
@peterblazejewicz 

> It will work with default esMduleInterop:

I know, but with CJS it's impossible export types (e.g. unions) which is useful in some cases.

> Not that this package has invalid package.json declaration and needs to be updated itself:

Already created [PR](https://github.com/piuccio/intl-unofficial-duration-unit-format/pull/27).